frontend/src/components/Product/ProductTitle/ProductTitle.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Text, theme } from '@fun-eat/design-system';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import { SvgIcon } from '@/components/Common';
+import productDetail from '@/mocks/data/productDetail.json';
+
+const ProductTitle = () => {
+  const { name, bookmark } = productDetail;
+  const [isBookmarked, setIsBookmarked] = useState(bookmark);
+
+  const toggleBookmark = () => {
+    setIsBookmarked(!isBookmarked);
+  };
+
+  return (
+    <ProductTitleContainer>
+      <ProductTitleWrapper>
+        <Button color="white" variant="filled">
+          <SvgIcon variant="arrow" color={theme.colors.gray5} width={15} height={15} />
+        </Button>
+        <Text
+          as="span"
+          css={`
+            margin-left: 10px;
+          `}
+        >
+          {name}
+        </Text>
+      </ProductTitleWrapper>
+      <Button
+        color="white"
+        variant="filled"
+        css={`
+          width: 2rem;
+        `}
+        onClick={toggleBookmark}
+      >
+        {isBookmarked ? (
+          <SvgIcon variant="bookmarkFilled" color={theme.colors.primary} />
+        ) : (
+          <SvgIcon variant="bookmark" color={theme.colors.gray5} />
+        )}
```

Comment:
```suggestion
          <SvgIcon variant={isBookmarked ? "bookmarkFilled" : "bookmark"} color={isBookMarked ? theme.colors.primary : theme.colors.gray5} />
```

컴포넌트 두 개보다 prop에 변화를 주는건 어떤가요?

Reply:
좋습니다~~ 수정 완!